var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -36,6 +36,10 @@ class NetlibLapack(Package):
     homepage = "http://www.netlib.org/lapack/"
     url = "http://www.netlib.org/lapack/lapack-3.5.0.tgz"
 
+    version('3.8.0', '96591affdbf58c450d45c1daa540dbd2',
+            url='http://www.netlib.org/lapack/lapack-3.8.0.tar.gz')
```

Comment:
It took me a while to get the `tar.gz` vs `tgz` difference. If they continue like this in future releases, we might think of an `url_for_version` method.